src/tldr.c

```diff
@@ -149,8 +154,8 @@ int main(int argc, char **argv) {
         return EXIT_SUCCESS;
     }
     if (verbose_flag && optind >= argc) {
-        print_version(argv[0]);
-        return EXIT_SUCCESS;
+        print_usage(argv[0]);
+        return EXIT_FAILURE;
```

Comment:
I assume this exits with code e.g. `1` instead of `0` when printing help? Other commands e.g. `ls`, `tar`, etc exit with code `0` when printing help, so we should do the same.

Unless I misunderstood this?

Reply:
Our client specification v1.4 suggests to use a [non-zero error code](https://github.com/tldr-pages/tldr/blob/main/CLIENT-SPECIFICATION.md) for finding pages. (https://github.com/tldr-pages/tldr/pull/4246)

Not sure if we should go with it here.

Reply:
\* Specifically, a client MUST exit with a non-zero exit code if the page could not be found anywhere.

Reply:
> I assume this exits with code e.g. `1` instead of `0` when printing help? Other commands e.g. `ls`, `tar`, etc exit with code `0` when printing help, so we should do the same.
\
Unless I misunderstood this?

This basically exits with `1` if invoked with `tldr` instead of `tldr --help`, which would exit with `0`.

In other words, if invoking `tldr` to display help on its own, rather than the user invoking it, the program would exit with `1`.

Example:
```sh
$ tldr > /dev/null; echo $?
1
$ tldr --help > /dev/null; echo $?
0
```

Reply:
> I assume this exits with code e.g. `1` instead of `0` when printing help? Other commands e.g. `ls`, `tar`, etc exit with code `0` when printing help, so we should do the same.
\
> Unless I misunderstood this?

This exits with `1` on a getopt error.

For example:
* `tldr -a`, which is an unknown argument, exits with `1` and displays a getopt error.
* `tldr -v` exits with `0` and displays the version.

Reply:
You can even test this by running these commands:

```sh
$ tldr -a 2> /dev/null; echo $?
1
$ tldr -v > /dev/null; echo $?
0
```

Reply:
Ah okay. So can you confirm that when you call e.g. `tldr --help` it exits with code `0`, @4G3NT?

Reply:
> Ah okay. So can you confirm that when you call e.g. `tldr --help` it exits with code `0`, @4G3NT?

Yes, it exits with code `0` because of this code:

```c
if (help_flag) {
    print_usage(argv[0]);
    return EXIT_SUCCESS;
}
```

Reply:
Very cool, many thanks!